### ISSUE TITLE
multiple definitions of 'aspect ratio' used in the NSC

### ISSUE THREAD

**Issue by egonw** [enanomapper/ontologies#141]
OECD/JRC uses a different definition currently used in the ENM ontology for 'aspect ratio.

Solution:

1. define a more general term that does not specify how it is calculated
2. have the current term that does define a calculation as child
3. add a new term for the OECD/JRC term

**Comment by dphilip** [comment]
I did some probing and we have this term that seems general:
http://bioportal.bioontology.org/ontologies/ENM/?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2Fnpo%23NPO_1365&jump_to_nav=true
and this term that is the result of computations and describes how it was calculated
http://bioportal.bioontology.org/ontologies/ENM/?p=classes&conceptid=http%3A%2F%2Fpurl.enanomapper.org%2Fonto%2FENM_8000064&jump_to_nav=true
Ideally I would like to keep both. Can you point to the OECD/JRC term?

Calling the latter "aspect ratio - image analysis descriptor" is an option?

**Comment by egonw** [comment]
Yes, we should keep both of these, but one is an (image) descriptor... the other has apaprently two definitions: largest/smallest and smallest/largest, which you know gives nanoQSAR issues :)

The plan is to make two subclasses of the NPO term, one for each definition...
